aws/terraform_install.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
+
+# This script updates the package list, installs Gnupg, downloads the Hashicorp signing key, adds the Hashicorp repository and installs Terraform.
+
+# Update Apt
+if (( $(date +%s) - $(stat -c %Y /var/lib/apt/periodic/update-success-stamp) > 24*60*60 ))
```

Comment:
A comment on what this is would be cool.

Reply:
This checks if apt has been updated in the last 24 hours. Seems some debate on what actually gets cached whenever apt is updated. Idk if this is the way but it seemed to have a lot of support on SO.